app/Database/migrations/2024_12_03_093851_add_sources_to_acars_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('acars', function (Blueprint $table) {
+            $table->string('source', 5)->nullable();
```

Comment:
You also want to set the encoding of this to `latin1` - otherwise, the default of `utf8mb4` is 4b/char

Reply:
I’m not sure if this changes anything, but we’re currently using `utf8mb3`. The upgrade to `utf8mb4` is planned within the Filament upgrade.

Reply:
Sure, that's just 3 bytes per char, but still more than we need

Reply:
Ah right. Well, that's 3 bytes/char, still more than we need

Reply:
@nabeelio @arthurpar06 When I attempt to do `$table->charset('latin1');`, it breaks all the tests. Is it possible to change the encoding of a table after it was created via migrations?

Reply:
> @nabeelio @arthurpar06 When I attempt to do `$table->charset('latin1');`, it breaks all the tests. Is it possible to change the encoding of a table after it was created via migrations?

You have to call `collation()` on just that column. [doc here](https://laravel.com/docs/10.x/migrations#column-modifiers)

Reply:
Fixed it. collation() throws the error, charset doesn't. Looks like we're good with this one.